pkgs/development/python-modules/magika/default.nix

```diff
@@ -0,0 +1,46 @@
+{ lib
+, buildPythonPackage
+, click
+, fetchPypi
+, numpy
+, onnxruntime
+, poetry-core
+, python-dotenv
+, pythonOlder
+, tabulate
+, tqdm
+}:
+
+buildPythonPackage rec {
+  pname = "magika";
+  version = "0.5.0";
+  pyproject = true;
+  disabled = pythonOlder "3.8";
+
+  src = fetchPypi {
```

Comment:
no need to change, but generally fetching from git repo is preffered over fetching from pypi

this is because pypi published packages ofthen don't have source code

Reply:
Thank you. I'll know in the future.